esphome/components/nextion/nextion.cpp

```diff
@@ -223,13 +225,42 @@ bool Nextion::read_until_ack_() {
         ESP_LOGD(TAG, "Got touch at x=%u y=%u type=%s", x, y, touch_event ? "PRESS" : "RELEASE");
         break;
       }
-      case 0x66:  // sendme page id
-      case 0x70:  // string variable data return
-      case 0x71:  // numeric variable data return
-      case 0x86:  // device automatically enters into sleep mode
-      case 0x87:  // device automatically wakes up
-      case 0x88:  // system successful start up
-      case 0x89:  // start SD card upgrade
+      case 0x66:    // sendme page id
+      case 0x70:    // string variable data return
+      case 0x71:    // numeric variable data return
+      case 0x86:    // device automatically enters into sleep mode
+      case 0x87:    // device automatically wakes up
+      case 0x88:    // system successful start up
+      case 0x89:    // start SD card upgrade
```

Comment:
Need to break here as otherwise `0x90` instructions will be called
```suggestion
      case 0x89:    // start SD card upgrade
        break;
```

Reply:
No need to break here, there is no action in 0x89 so we just skip over to 0x90 case

Reply:
I think this break should be there. We don't want 0x90 code to be executed for the previous commands

Reply:
Yes you are right, I will insert the break.